appinventor/blocklyeditor/src/workspace_svg.js

```diff
@@ -568,6 +568,26 @@ Blockly.WorkspaceSvg.prototype.customContextMenu = function(menuOptions) {
   };
   menuOptions.splice(3, 0, exportOption);
 
+  //Hide workspace SVG elements backpack, zoom, and trashcan
```

Comment:
I'm a little bit worried about having so many options on the context menu. Some others, like show/hide all comments, make sense because either is valid if there is a mix. However, for this change, it can only ever be one state or the other. Rather than have two options, I think we should have a single option and have it either be show or hide based on the current state.